models/models.js

```diff
@@ -19,3 +19,12 @@ exports.selectArticleByID = (article_id) => {
             }
         })
 }
+
+exports.selectAllArticles = () => {
+    console.log('models');
+    return db
+        .query('SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id ORDER BY created_at DESC;')
```

Comment:
Would make more sense if the comment_count is a number rather than a string. To make it a number, just need to add `::INT` before `AS`. It'll help when sorting the articles by comment_count